Packs/LINENotify/Integrations/LINENotify/LINENotify.yml

```diff
@@ -0,0 +1,34 @@
+category: Utilities
+commonfields:
+  id: LINENotify
+  version: -1
+configuration:
+- display: Token of LINE Group
+  name: apitoken
+  required: true
+  type: 4
+contentitemexportablefields:
+  contentitemfields:
+    fromServerVersion: ""
+    itemVersion: ""
+    packID: 1d1a0d4c-63fb-45ea-8f98-276b17bae933
+    propagationLabels: []
+    toServerVersion: ""
+description: |-
```

Comment:
Please fix the description here according to the LINENotify

Reply:
This is the LINE API Integration for sending message to LINE Group.  Before you use this integration, you need to get a LINE Group token from your account. 
The easy way to do this is
1. Login nhttps://notify-bot.line.me/my/ with your username/password of line
2. Click 'your name' in top-right, and select "My page" 
3. Click 'Generate Token' ,  and then input a token name and select a chat group.
4. Then click 'Generate token', then you will see a token with you.  Make sure you copy the token string and keep it.

After that, you could go back to LINE, and invite 'LINE Notify' bot into your chat group.  
The bot (LINE Notify) will send/forward your message to the LINE group while you trigger it from XSOAR integration.